.travis.yml

```diff
@@ -0,0 +1,15 @@
+os: osx
+language: objective-c
+osx_image: xcode8.3
+node_js: 5.1.1
+
+before_install:
+  - cd server
+
+install:
+  - npm install
+
+script:
```

Comment:
Cool! I have vague memories of me trying Travis in the past but also not getting browser tests to run. Some pointers might be: I'm using `tape-run` which uses `browser-run` which uses electron - and all on somewhat outdated versions I assume. 

Btw if you just want to run local tests you can just call `npm run test-local`. No build step should be needed